spec/abstractops.html

```diff
@@ -494,7 +492,14 @@ <h1>ToSmallestTemporalDurationUnit ( _normalizedOptions_, _disallowedUnits_, _fa
   <emu-clause id="sec-temporal-totemporaldurationtotalunit" aoid="ToTemporalDurationTotalUnit">
     <h1>ToTemporalDurationTotalUnit ( _normalizedOptions_ )</h1>
     <emu-alg>
-      1. Let _unit_ be ? GetOption(_normalizedOptions_, *"unit"*, *"string"*, « *"year"*, *"years"*, *"month"*, *"months"*, *"weeks"*, *"day"*, *"days"*, *"hour"*, *"hours"*, *"minute"*, *"minutes"*, *"second"*, *"seconds"*, *"millisecond"*, *"milliseconds"*, *"microsecond"*, *"microseconds"*, *"nanosecond"*, *"nanoseconds"* », *undefined*).
+      1. Return ? ToTemporalDurationUnit(_normalizedOptions_, *"unit"*, *undefined*).
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-temporal-totemporaldurationunit" aoid="ToTemporalDurationUnit">
+    <h1>ToTemporalDurationUnit ( _normalizedOptions_, _key_, _fallback_ )</h1>
+    <emu-alg>
```

Comment:
I'm trying to make sure we add a `<p>` with a blurb about each abstract operation going forward, and possibly information about the types of the arguments. Would you mind adding one here?

Reply:
How about this, similar to the ones I wrote in #1509?
```suggestion
    <p>
      The abstract operation ToTemporalDurationUnit consults the _key_ property of the Object _normalizedOptions_, and checks that it is a valid unit.
      It returns _fallback_ if the property is not present.
    </p>
    <emu-alg>
```